### DISCUSSION TITLE
How do I check the length of a relation in a custom api update rule?

### DISCUSSION THREAD

**Discussion by JosephTTD** [pocketbase/pocketbase#1582] [Q&A] [answered]
I have a relation type called participants in one of my collections with a max select of 2. The collection in which participants is can be updated by any verified user but only participant field in general. I tried adding a costume rule where if max select is met then it can't be updated, but with everything I've tried I've not gotten the desired response. I've tried `participants.length < 2`, `@request.data.participants.length < 2`, `participants < 2`. How do I check the length or prevent updates if max selects are met?

**Comment by ganigeorgiev** [comment, marked as answer]
With the recent [v0.11.0](https://github.com/pocketbase/pocketbase/releases/tag/v0.11.0) we now have support for `:length` modifier that you could use as part of a `@request.data.*` or regular collection rel field.

`@request.data.participants:length < 2` will allow **submitting** no more than 1 participant.

`participants:length < 2` will satisfy only those record(s) that has less than 2 participants.

_The modifiers will be documented properly with the planned docs changes in v0.13/v0.14._

**Reply by ganigeorgiev** [reply]
_Also note that you don't need the above if you want a global `relation` length validator (aka. will be applied also for admins) by setting `Max Select` to 2 in the corresponding field options panel._

**Reply by JosephTTD** [reply]
Thank you, I'll give it a go, I'm still using v0.8, so I'm going to need to update.

**Reply by JosephTTD** [reply]
Thanks, upgrading the dashboard solved my issue 👍
